hr_expense_invoice/models/account_payment.py

```diff
@@ -1,21 +0,0 @@
-# Copyright 2019 Kitti U. <kittiu@ecosoft.co.th>
-# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
-from odoo import _, models
-from odoo.exceptions import ValidationError
-
-
-class AccountPayment(models.Model):
```

Comment:
@pedrobaeza because this function on core odoo not used and it change to `post` on v13+

V12: https://github.com/odoo/odoo/blob/12.0/addons/account/views/account_payment_view.xml#L290

V13: https://github.com/odoo/odoo/blob/13.0/addons/account/views/account_payment_view.xml#L313

and V14 remove view `view_account_payment_invoice_form` already

Reply:
But this code was for avoiding an expense to be paid through payments, so I don't see the relation.